kernel/arch/dreamcast/include/dc/video.h

```diff
@@ -279,14 +299,19 @@ void vid_flip(int fb);
     
     \return                 Old border color value (RGB888)
 */
-uint32 vid_border_color(int r, int g, int b);
+uint32_t vid_border_color(int r, int g, int b);
```

Comment:
```suggestion
uint32_t vid_border_color(uint8_t r, uint8_t g, uint8_t b);
```
I feel like we might as well fix this while we're at it... The only thing that makes sense for RGB888 is for each channel to be `uinht8_t`, which makes it easier on the user.

Reply:
Good catch. Updated.